examples/resource-path-parameters/resource_path_parameters.md

```diff
@@ -4,6 +4,10 @@ Path segments can be treated as parameters in Ballerina.
 
 ::: code resource_path_parameters.bal :::
 
-::: out resource_path_parameters.client.out :::
+Run the service using the `bal run` command.
```

Comment:
```suggestion
Run the service using the `bal run` command.
```
The command should be added to the `.out` file. Please check and fix all occurrences.

Reply:
Checked and updated the relevant out files